src/algopy_testing/context.py

```diff
@@ -1047,20 +1047,23 @@ def any_transaction(  # type: ignore[misc]
 
         return new_txn
 
-    def get_box(self, name: algopy.Bytes | bytes) -> algopy.Bytes:
+    def does_box_exist(self, name: algopy.Bytes | bytes) -> bool:
```

Comment:
@boblat can you add a few sentences to the docs on https://github.com/algorandfoundation/algorand-python-testing/blob/main/docs/coverage.md and https://github.com/algorandfoundation/algorand-python-testing/blob/main/docs/usage.md 

The coverage.md changes can be a matter of describing that higher level box interface is 'Emulated' meaning that it has an abstraction in context manager that mimics avm. And the usage.md can be just an example of using the boxes.

Reply:
I've added some notes to the docs.
[add notes for higher level Box interfaces to docs](https://github.com/algorandfoundation/algorand-python-testing/pull/3/commits/7565f8debfaed2e45c3a59eb8625d98c2b2c03b4)